test/create_env_cluster_topics_test.go

```diff
@@ -15,91 +19,250 @@ import (
+		}
+		a.Equal("BigQuerySink", actualBigQueryConnectorClass, "BigQuery connector class is not 'BigQuerySink'.")
+		log.Printf("Validated BigQuery sink connector class 'BigQuerySink', name '%s' and status 'RUNNING' successfully.", bigQueryConnectorName)
 
-		output = terraform.Output(t, runOptions, "environment_name")
-		a.True(strings.Contains(output, "labs-environment-"+runID))
+		// Validate GCS Sink Connector
+		gcsConnectorOutput := runCLICommand(t, "confluent", "connect", "cluster", "describe", gcsSinkConnectorID, "-o", "json")
+		var gcsConnectorDetails map[string]interface{}
+		a.NoError(json.Unmarshal([]byte(gcsConnectorOutput), &gcsConnectorDetails), "Failed to parse GCS connector output")
+		a.Equal("RUNNING", gcsConnectorDetails["connector"].(map[string]interface{})["status"], "GCS connector is not running.")
+		a.Equal(gcsConnectorName, gcsConnectorDetails["connector"].(map[string]interface{})["name"], "GCS connector name in not correct.")
 
-		output = terraform.Output(t, runOptions, "environment_id")
-		a.NotEmpty(output)
+		// GCS Connector class
+		var actualGCSConnectorClass string
+		for _, config := range gcsConnectorDetails["configs"].([]interface{}) {
+			configMap := config.(map[string]interface{})
+			if configMap["config"] == "connector.class" {
+				actualGCSConnectorClass = configMap["value"].(string)
+			}
+		}
+		a.Equal("GcsSink", actualGCSConnectorClass, "GCS Connector class is not 'GcsSink'")
+		log.Printf("Validated GCS sink connector class 'GcsSink', name '%s' and status 'RUNNING' successfully.", gcsConnectorName)
 
-		output = terraform.Output(t, runOptions, "kafka_cluster_basic_name")
-		a.True(strings.Contains(output, "labs-kafka-cluster-"))
+		// Validate Topic Service Account name
+		log.Println("Validating service account name...")
+		serviceAccountsOutput := runCLICommand(t, "confluent", "iam", "service-account", "list", "-o", "json")
+		var serviceAccounts []map[string]interface{}
+		a.NoError(json.Unmarshal([]byte(serviceAccountsOutput), &serviceAccounts), "Failed to parse service accounts output")
 
-		output = terraform.Output(t, runOptions, "topic_service_account_key")
-		a.NotEmpty(output)
+		for _, account := range serviceAccounts {
+			if account["name"] == adminServiceAccountName {
+				a.Equal(adminServiceAccountName, account["name"], "Admin service account name is not crrect.")
+				log.Printf("Admin service account '%s' validated successfully.", adminServiceAccountName)
+			} else if account["name"] == topicServiceAccountName {
+				a.Equal(topicServiceAccountName, account["name"], "Topic service account name is not crrect.")
+				log.Printf("Topic Service Account '%s' validated successfully.", topicServiceAccountName)
+			} else if account["name"] == connectorServiceAccountName {
+				a.Equal(connectorServiceAccountName, account["name"], "Connector service sccount name is not correct.")
+				log.Printf("Connector service account '%s' validated successfully.", connectorServiceAccountName)
+			}
```

Comment:
You'll want an else here to fail the test if you have an account that was not one of the expected three.